### PR TITLE
Fix Customer, Tradie Components

### PR DESCRIPTION
**Pull request by Fumicode01**
## Rationale

This is for testing the git workflow

## Advice for Reviewers & Testing Notes

- Explain how your change should be reviewed, and where reviewers should direct their attention.
- Explain (where relevant) how your change can be tested.

## Screenshots:

- Add a screenshot or two of your changes, where applicable.

## Task Name

- Issue Title

## Linting Checklist
- [ ] No commented code
- [ ] Code Formatted nicely (Prettier)
- [ ] PR your own code before you assign reviewers

### CODE REVIEW

**Review by kimchirice — CHANGES_REQUESTED** [review]
~~it's broken on my local repo after fetching the PR~~ 

before stepping into ~~troubleshooting~~ fetching the PR and reviewing, could you fill in a few details of the PR? 
````
> ## Rationale
> This is for testing the git workflow
> 
> ## Advice for Reviewers & Testing Notes
> * Explain how your change should be reviewed, and where reviewers should direct their attention.
> * Explain (where relevant) how your change can be tested.
> 
> ## Screenshots:
> * Add a screenshot or two of your changes, where applicable.
> 
> ## Task Name
> * Issue Title
> 
> ## Linting Checklist
> * [ ]  No commented code
> * [ ]  Code Formatted nicely (Prettier)
> * [ ]  PR your own code before you assign reviewers
````

or at least a screenshot if not a recording clip 

@engramar maybe we could update the PR template, rephrase "This is for testing the git workflow" to "please take time to provide detail information of the pull request listed below"

**Comment by Fumicode01** [comment]
> ## Rationale
> This is for testing the git workflow
> 
> ## Advice for Reviewers & Testing Notes
> * Explain how your change should be reviewed, and where reviewers should direct their attention.
> * Explain (where relevant) how your change can be tested.
> 
> ## Screenshots:
> * Add a screenshot or two of your changes, where applicable.
> 
Screen Shot 2021-06-15 at 3 18 42 pm
Screen Shot 2021-06-15 at 3 18 17 pm


> ## Task Name
> * Add Customer and Tradie SignUp/Login page in sidebar with router
> 
> ## Linting Checklist
> * [ ]  No commented code
> * [ ]  Code Formatted nicely (Prettier)
> * [ ]  PR your own code before you assign reviewers


I edited on master branch from Akari's work.
I just added customer and Tradie SignUp and Login page which can jump from sidebar by Router.
In my own repo, it is working.

File: src/App.js [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
       <RightSideBar />
       <Switch>
-        <Route exact={true} path='/' component={NewInventory} />
+        <Route exact={true} path='/' component={SignUp} />
         <Route exact path='/login' component={TradieLogin} />
+        <Route exact path='/customer/signup' component={CustomerSignUp} />
+        <Route exact path='/customer/login' component={CustomerLogin} />
+        <Route exact path='/tradie/signup' component={TradieSignUp} />
+        <Route exact path='/tradie/login' component={TradieLogin} />
+
```
Comment (kimchirice) [inline src/App.js:25]: not sure about the path="/",  my understanding is, 

-  by default, signIn page should be processing TradieSignIn component
- "/customer/signup" and "/customer/login" should Not be public routes

**Comment by Fumicode01** [comment]
Actually, I should not add these on sidebar.
I added due to editing these area without changes in app.js everytime to switch components.
I will delete these soon after I fix some minor errors for me.

**Comment by Fumicode01** [comment]
I am going to close this pull request once.

File: src/components/Forms/useForm.js [hunk 2]
```diff
@@ -32,9 +34,31 @@ const useForm = validate => {
...
+            confirmpassword: values.confirmpassword,
+            address: values.address,
+            state: values.state,
+            postcode: values.postcode,
+            showPassword: false,
+            licence:values.licence,
+            role:values.role,
+            categories:values.categories,
+            description: values.description,
+
```
Comment (kimchirice) [inline src/components/Forms/useForm.js:57]: looking up [the swagger](http://taskstech2.pythonanywhere.com/swagger/#/users/post_users_customer)

- create a customer
{
  "email": "string",
  "password": "string",
  "first_name": "string",
  "last_name": "string",
  "phone": "string",
  "address": "string"
}
- create tradeperson
{
  "email": "string",
  "password": "string",
  "first_name": "string",
  "last_name": "string",
  "description": "string",
  "phone": "string"
}

maybe change the payload of post req,  ONLY including those fields at the time of creating a new tradeperson and customer

File: src/components/Forms/TradieSignUp.js [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+            Tradies Sign up
+          </Typography>
+          <form className={classes.form}  onSubmit={handleSubmit}>
+          <Grid container spacing={3}>
+          <Grid container>
+              {/* <Grid item xs>
+                <Link href="#" variant="body2">
+                  Forgot password?
+                </Link>
+              </Grid> */}
```
Comment (kimchirice) [inline src/components/Forms/TradieSignUp.js:67]: pls delete uncessary comments

Comment (Fumicode01) [reply src/components/Forms/TradieSignUp.js:67]: Sure.
I will clean up unenessary comments and parts of post request.

**Pull request closed without merging**